util/mp-functions.xsl

```diff
@@ -352,6 +354,7 @@
                 <xsl:if test="$periode/@value">gedurende <xsl:value-of select="concat($periode/@value, ' ', nwf:unit-string($periode/@value, $periode/@unit))"/></xsl:if>
```

Comment:
Shouldn't this be a check on `$periode[@value]` (i.e. [..] instead of a /)?

Reply:
Done